ejb3/src/main/resources/org/jboss/as/ejb3/subsystem/LocalDescriptions.properties

```diff
@@ -24,7 +24,7 @@ ejb3.default-security-domain=The default security domain that will be used for J
 ejb3.default-missing-method-permissions-deny-access=If this is set to true then methods on an Jakarta Enterprise Beans with a security domain specified or with other methods with security metadata will have an implicit @DenyAll unless other security metadata is present
 ejb3.disable-default-ejb-permissions=This deprecated attribute has no effect and will be removed in a future release; it may never be set to a "false" value
 ejb3.disable-default-ejb-permissions.deprecated=Adding default permissions to Jakarta Enterprise Beans deployments is no longer supported and this configuration attribute will be removed in a future release
-ejb3.enable-graceful-txn-shutdown=Enabling txn graceful shutdown will make the server wait for active Jakarta Enterprise Beans-related transactions to complete before suspending. For that reason, if the server is running on a cluster, the suspending cluster node may receive ejb requests until all active transactions are complete. To avoid this behavior, omit this tag.
+ejb3.enable-graceful-txn-shutdown=Enabling txn graceful shutdown will make the server wait for active remote Jakarta Enterprise Beans-related transactions to complete before suspending. For that reason, if the server is running on a cluster, the suspending cluster node may receive Jakarta Enterprise Beans requests until all active remote transactions are complete. To avoid this behavior, omit this tag. This attribute has no effect on local Jakarta Enterprise Beans-related transactions.
```

Comment:
Further improvement:

s/omit this tag/leave this attribute undefined or set it to 'false'/g

'Tag' is a presumably reference to xml but this string isn't part of an xml schema.

Reply:
Might be indeed a good idea to fix the description as well. I'll pass this to frainone as I'm waiting for her approval of these changes. Thank you @bstansberry